crates/wash-cli/src/bin/wash.rs

```diff
@@ -462,7 +463,11 @@ async fn main() {
         CliCommand::Wit(wit_cli) => wit_cli.run().await,
     };
 
-    std::process::exit(match res {
+    // Use buffered writes to stdout preventing a broken pipe error in case this program has been
+    // piped to another program (e.g. 'wash dev | jq') and CTRL^C has been pressed.
+    let mut stdout_buf = BufWriter::new(stdout().lock());
```

Comment:
LG overall. Is there any way we can test that we're using buffered writes? Testing for the actual bug looks messy but testing the buffered writes should give us coverage

Reply:
That looks great @michelm -- if you want to add that to this PR you can go ahead, or we could make a follow up issue?

Reply:
If possible I would like to add the integration test in a follow-up issue. If so, please feel free to assign my name to it. Otherwise, let me know and I will add it to the current PR.

Reply:
Add it in this PR.

Reply:
Hey @michelm going to merge this! I'll make an issue and assign you :)